app/assets/stylesheets/v3/layout_components/grids.scss

```diff
@@ -0,0 +1,23 @@
+.grid {
+  display: grid;
+  grid-gap: 2rem;
+
+  &--2 {
+    @extend .grid;
+    grid-template-columns: 1fr 1fr;
+
+    @include breakpoint(small down) {
+      grid-template-columns: 1fr;
+    }
+  }
+
+  &--3 {
+    @extend .grid;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  &--4 {
+    @extend .grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
+}
```

Comment:
I see we created this file, but it isn't being used in your GK PR. Can we remove it?